src/test/e2e/PageObjects/HomePage.ts

```diff
@@ -0,0 +1,31 @@
+import { Page } from 'puppeteer';
+
+const helpers = require('../Helpers/Selectors');
+
+class HomePage {
+  async OpenHomePage(page: Page): Promise<any> {
+    await page.goto(helpers.baseUrl, {waitUntil: 'domcontentloaded'});
+
+    const pageTitle = await page.$eval(helpers.MainHeader, (e: Element) => e.textContent);
+    expect(pageTitle).toBe('Find a court or tribunal listing');
```

Comment:
we should have the tests on the test page, these page objects should return pages we are on and the evaluating be done on the object